tsdb/shard.go

```diff
@@ -16,16 +21,41 @@ import (
 	"github.com/gogo/protobuf/proto"
 )
 
-// Shard represents a self-contained time series database. An inverted index of the measurement and tag data is
-// kept along with the raw time series data. Data can be split across many shards. The query engine in TSDB
-// is responsible for combining the output of many shards into a single query result.
+const (
+	// DefaultMaxWALSize is the default size of the WAL before it is flushed.
+	DefaultMaxWALSize = 10 * 1024 * 1024 // 10MB
```

Comment:
Not a blocker for this PR, but we'll need to make this configurable through `tsdb/config.go`.